WebApiCoreSeed.WebApi/Controllers/Dtos/ErrorDto.cs

```diff
@@ -0,0 +1,18 @@
+namespace WebApiCoreSeed.WebApi.Controllers.Dtos
+{
+    /// <summary>
+    /// Error data transfer object
+    /// </summary>
+    internal class ErrorDto
+    {
+        public ErrorDto(string error)
+        {
+            Error = error;
+        }
+
+        /// <summary>
+        /// Error message
+        /// </summary>
+        public string Error { get; private set; }
```

Comment:
It is not necessary to specify `private set;`. I mean, if _Error_ is only set in the constructor `public string Error { get; }` will also work.

Reply:
Old habits, ty